protractor/spec.js

```diff
@@ -0,0 +1,50 @@
+const { browser, element } = require('protractor');
+// const jasmine = require('jasmine');
+//const expect = jasmine.expect;
+
+describe('angulario homepage', function () {
+    beforeEach(function () {
+        browser.get('http://angular.io/');
+    });
+    it('should show the search results', function () {
+        element(by.xpath('//input[@type=\'search\']')).sendKeys('What is Angular');
+        const searchResult = element(by.className('search-results'));
```

Comment:
You can declare all the locators before the tests, because in this way after any chnage you will go and search in locators in test cases